brokerbot.go

```diff
@@ -141,19 +141,28 @@ func handleMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
 	expandedString := messagelib.ExpandAliases(userInput)
 	tickers := messagelib.DedupeTickerStrings(strings.Split(expandedString, " "))
 
+	cryptoContext, cancelFunc := context.WithCancel(ctx)
```

Comment:
Cancellable contexts are useful when using concurrency. None of this code is concurrent so you can omit this entirely